pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -100,11 +100,11 @@ class Build {
     def runTests() {
         def testStages = [:]
         List testList = []
-        
+
         if (buildConfig.VARIANT == "hotspot-jfr") {
```

Comment:
Can you also add corretto to this if so we do not run sanity.external tests (which pull hs/j9 docker images from docker hub repo to test with by default).  If we want to include this set for corretto, we will have to add special test targets that mount the SDK using EXTRA_DOCKER_ARGS="-v $TEST_JDK_HOME:/opt/java/openjdk".  Since we don't have those targets at the moment, let's not call this set of tests.

Reply:
done